tests/Unit/ParallelAlgorithms/LinearSolver/Schwarz/Test_SchwarzAlgorithm.yaml

```diff
@@ -20,6 +20,15 @@ DomainCreator:
     InitialRefinement: [1]
     InitialGridPoints: [3]
     TimeDependence: None
+    BoundaryConditions:
+      LowerBoundary:
+        TestBoundaryCondition:
+          Direction: lower-xi
+          BlockId: 0
+      UpperBoundary:
+        TestBoundaryCondition:
+          Direction: upper-xi
+          BlockId: 0
```

Comment:
I'm fine with making these changes, though you could get around that by specifying the test-system without boundary conditions. We may end up with a way of avoiding dummy boundary conditions for things like this or ExportCoordinates or ApplyObservers, so it might be easier to do what you did for ExportCoordinates and just add the empty system here. Up to you.

Reply:
I made the change anticipating you'd want to add boundary conditions into these tests afterward :)

Reply:
Ah I see. Would you mind going with the empty-system approach for now then? The linear solver tests won't use boundary conditions because they work with hard-coded matrices.

Reply:
sure!